src/components/pages/Contacts/index.tsx

```diff
@@ -306,6 +364,16 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
                       </div>
                     ))}
                   </td>
+                  <td>
+                    <Button
+                      title={'send ' + contact.first_name + ' a message'}
```

Comment:
I think message is a bit ccnfusign.

I'd just say "Mail" instead

Reply:
changed to 'send mail to' + name